src/jira/client/jira-client.ts

```diff
@@ -356,6 +356,14 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		appProperties: {
+			create: (appKey: string, isConfiguredState: string) =>
```

Comment:
/nit
I don't think `appKey` gonna be different upon each time this function is called (within same env). So maybe easier to read the code if we call `getAppKey()` within this function instead of receive it as an arg.